app/views/govuk_publishing_components/components/_cards.html.erb

```diff
@@ -0,0 +1,59 @@
+<%
+  heading ||= nil
+  items ||= nil
+  sub_heading_level = sub_heading_level || 3
```

Comment:
I mentioned this same issue in the PR description ('Things to note'). It seems like calculating the heading level could be a future improvement because the `get_heading_level` helper doesn't return an integer but a string so we'd need to do some splitting of strings. It seems like that the logic for that would be better either as a shared helper or maybe returned by `get_heading_level` somehow. 

Or let me know if I have in fact overlooked a way to easily get the integer from `get_heading_level`, I didn't too deeply into it.

Reply:
Ah yes I see, the shared_helper returns a string like `h2` rather than an integer. This might be too complicated for this initial version of the component then. A future improvement might be to modify the shared_helper something like this:

```
def get_heading_level
  return [*1..6].include?(@heading_level) ? "h#{@heading_level}" : "h2" unless @heading_level.zero?

  "span"
end

def get_subheading_number
  @heading_level + 1 unless get_heading_level == "span"
end
```